src/ctapipe_io_nectarcam/__init__.py

```diff
@@ -92,6 +92,123 @@ class PixelStatus(IntFlag):
     num_mirror_tiles=86,  # Garczarczyk 2017
 )
 
+def module_central(geometry):
+    '''
+    Input:
+        camera geometry  
+            e.g. `source.subarray.tel[10].camera.geometry`
+    Returns:
+        pix_mid_module
+            list of pixels at the centre of each module (sorted)
+        
+    Requires the geometry to have pixel_x and pixel_y, and to have 7-pixel modules.
+    Neighbours gets lazy-loaded by the main ctapipe function in camera.geometry.
+    
+    Find central pixels of all the modules, as follows:
+
+    Finds the most distant PMT and which still has 6 neighbours, 
+      which must be the central pixel of a module.
+    Then elimates that module and repeats, until no modules left.
+
+    '''
+    
+    import numpy as np
+    
+    # Bug out right away if the number of pixels is not divisible by 7 (for 7-pixel modules)
+    if geometry.n_pixels%7 != 0:
+        raise ValueError("n_pixels is not divisible by 7, so not full 7-pixel modules. \n"+
+            "... Incompatible with expected NectarCAM geometry.")
+    
+    pix_mid_module = []
+    
+    pixel_dist = np.hypot(geometry.pix_x,geometry.pix_x).value
```

Comment:
Shouldn't it be:
```
    pixel_dist = np.hypot(geometry.pix_x,geometry.pix_y).value
```
instead ?
Otherwise, it should be equivalent (and maybe even faster) to use:
```
np.sqrt(2.) * geometry.pix_x.value
```

Reply:
Yes, it would be great if you could provide unit tests for the added methods.

Reply:
Done.